Cargo.toml

```diff
@@ -398,5 +398,14 @@ subxt = { version = "0.39.0", default-features = false }
 subxt-core = { version = "0.39.0", default-features = false }
 subxt-signer = { version = "0.39.0", default-features = false }
 
+# Blueprint
+blueprint-manager = { default-features = false, git = "https://github.com/tangle-network/blueprint", branch = "polkadot-stable2407" }
+blueprint-runner = { default-features = false, git = "https://github.com/tangle-network/blueprint", branch = "polkadot-stable2407" }
+blueprint-keystore = { default-features = false, git = "https://github.com/tangle-network/blueprint", branch = "polkadot-stable2407" }
+# --
+# blueprint-manager = { default-features = false, path = "../gadget/crates/manager" }
```

Comment:
Delete this?

Reply:
It was left for me for quickly testing things locally. I will remove it in next PRs.